.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM oven/bun:debian
```

Comment:
Can you try running `bun run --bun payload info` inside the dev container? I think this is going to fail without `node`, see https://github.com/payloadcms/payload/issues/15015 for details.